src/screens/login/Login.jsx

```diff
@@ -66,10 +72,13 @@ const Login = ({ navigation }) => {
         autoCapitalize="none"
       />
 
-      <Typography id="PASSWORD" style={styles.textFontInput} />
+      <Typography
+        id="PASSWORD"
+        style={[globalStyles.h2Text, globalStyles.inputHeader]}
```

Comment:
```suggestion
        style={globalStyles.h2Text, globalStyles.inputHeader}
```
Same issue as above ^